client/auth/login.jsx

```diff
@@ -67,16 +67,36 @@ module.exports = React.createClass( {
 
 	componentWillUnmount: function() {
 		AuthStore.off( 'change', this.refreshData );
+		clearTimeout( this.pollTimeout );
+		this.pollTimeout = null;
 	},
 
 	refreshData: function() {
 		this.setState( AuthStore.get() );
```

Comment:
This can sometimes be misleading. I realize this wasn't introduced in this PR, but in general it's nice to store these implicit states inside a named parameter.

```js
this.setState( { data: AuthStore.get() } );
```

in doing that, we can contain the names and prevent surprises and name-clashes. Not sure if this would be an easy switch-out here or not.